### PR TITLE
update references to public key with verification material

### PR DESCRIPTION
**Pull request by nicksteele**
***
<a href="https://pr-preview.s3.amazonaws.com/nicksteele/vc-data-model/pull/1414.html" title="Last updated on Jan 18, 2024, 7:15 PM UTC (b8d105a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1414/2980929...nicksteele:b8d105a.html" title="Last updated on Jan 18, 2024, 7:15 PM UTC (b8d105a)">Diff</a>

### CODE REVIEW

**Comment by nicksteele** [comment]
Sorry, branch management is hard

File: index.html [hunk 1]
```diff
@@ -734,9 +734,9 @@ <h3>Credentials</h3>
 A [=credential=] is a set of one or more [=claims=] made by the same
 [=entity=]. [=Credentials=] might also include an identifier and metadata
 to describe properties of the [=credential=], such as the
-[=issuer=], the validity date and time period, a representative image, a
-public key to use for [=verification=] purposes, the revocation mechanism,
-and so on. The metadata might be signed by the [=issuer=]. A
+[=issuer=], the validity date and time period, a representative image, 
+verification material to use for [=verification=] purposes, the revocation
```
Comment (TallTed) [inline index.html:738]:
```suggestion
[=verification material=], the revocation 
```

File: index.html [hunk 2]
```diff
@@ -5787,7 +5787,7 @@ <h3>Key Management</h3>
 provides recommendations for protective measures, harm reduction, and
 revocation. Although this section focuses primarily on the security of the
 private signing key, [[NIST-SP-800-57-Part-1]] also highly recommends
-confirmation of the validity of all public keys before using them.
+confirmation of the validity of all verification material before using it.
```
Comment (TallTed) [inline index.html:5790]:
```suggestion
confirmation of the validity of all [=verification material=] before using it.
```

File: index.html [hunk 3]
```diff
@@ -6339,8 +6339,8 @@ <h3>Credential Subject</h3>
 `credentialSubject` is expected to identify a [=subject=] to the
 [=verifier=]. If the [=holder=] is also the [=subject=], then
 the [=verifier=] could authenticate the [=holder=] if they have
-public key metadata related to the [=holder=]. The [=verifier=] could then
-authenticate the [=holder=] using a signature generated by the [=holder=]
+verification material metadata related to the [=holder=]. The [=verifier=]
```
Comment (TallTed) [inline index.html:6342]:
```suggestion
[=verification=] metadata related to the [=holder=]. The [=verifier=]
```

File: index.html [hunk 4]
```diff
@@ -6390,11 +6390,10 @@ <h4>Holder</h4>
 Often relevant metadata about the [=holder=], as identified by the value of
 the `holder` [=property=], is available to, or retrievable by, the
 [=verifier=]. For example, a [=holder=] can publish information containing
-the verification material used to secure
-[=verifiable presentations=]. This metadata is expected to be used when
-checking proofs on [=verifiable presentations=]. Some cryptographic
-identifiers contain all necessary metadata in the identifier itself. In those
-cases, no additional metadata is required. Other identifiers use verifiable data
+the verification material used to secure [=verifiable presentations=]. This
```
Comment (TallTed) [inline index.html:6393]:
```suggestion
the [=verification material=] used to secure [=verifiable presentations=]. This 
```

File: index.html [hunk 5]
```diff
@@ -6484,8 +6483,8 @@ <h3>Proofs (Signatures)</h3>
 
         <ul>
           <li>
-Acceptably recent metadata regarding the public key associated with the
-signature is available. For example, the metadata might include
+Acceptably recent metadata regarding the verification material associated
```
Comment (TallTed) [inline index.html:6486]:
```suggestion
Acceptably recent metadata regarding the [=verification material=] associated 
```

**Review by TallTed — CHANGES_REQUESTED** [review]
tweaked language and crosslinks

**Review by dlongley — APPROVED** [review]

File: terms.html [hunk 6]
```diff
@@ -198,6 +198,11 @@
 <a>verifiable presentation</a> for processing. Other specifications might refer
 to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
   </dd>
+  <dt><dfn data-lt="verification material">verification material</dfn></dt>
+  <dd>
+    Cryptographic information could be a public key or any other cryptographic data
```
Comment (dlongley) [inline terms.html:203]: Hmm, perhaps a slight tweak here to be more flexible:

```suggestion
    Information that could be a cryptographic public key or any other data 
```

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**